src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -73,14 +74,16 @@ public static function register($mode = 0)
         }
 
         self::$utilPrefix = class_exists('PHPUnit_Util_ErrorHandler') ? 'PHPUnit_Util_' : 'PHPUnit\Util\\';
+        self::$isHandlerInvokable = method_exists(self::$utilPrefix.'ErrorHandler', '__invoke');
 
         $handler = new self();
         $oldErrorHandler = set_error_handler([$handler, 'handleError']);
 
         if (null !== $oldErrorHandler) {
             restore_error_handler();
 
-            if ([self::$utilPrefix.'ErrorHandler', 'handleError'] === $oldErrorHandler) {
+            $handlerMethod = self::$isHandlerInvokable ? '__invoke' : 'handleError';
+            if ([self::$utilPrefix.'ErrorHandler', $handlerMethod] === $oldErrorHandler) {
```

Comment:
if the previous handler is the PhpUnit one, then `$oldErrorHandler` will contains an instance of `PHPUnit\Util\ErrorHandler`